account_move_ordering_contacts/models/account_move.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    ordering_contact_ids = fields.Many2many(
+        comodel_name="res.partner",
+        relation="account_move_ordering_contact_rel",
+        column1="account_move_id",
+        column2="res_partner_id",
+        compute="_compute_ordering_contact_ids",
+        store=True,
+        string="Ordering Contacts",
+    )
+
+    @api.depends("move_type", "invoice_line_ids.sale_line_ids.order_id.partner_id")
```

Comment:
are you sure this deeply nested relation depends will work?

Reply:
@simahawk ran some tests, it works as supposed. However, I found that the field is left empty upon module installation, because of this: https://github.com/odoo/odoo/blob/18362722/odoo/models.py#L2598-L2605
Since method `update_db()` returns `None` if the field column type is not defined, and we're initializing a `Many2many` field, the field is never set as "to recompute". Therefore, it won't be recomputed upon existing records when the module is installed.
Any idea about how to fix this? I was thinking about overriding the `init` method, check whether the table `account_move_original_partner_rel` exists and whether it's empty or not; if exists but it's empty, then the field should be recomputed. Could it work?

Reply:
I'd go for a post init hook